### PR TITLE
Document the api endpoints you are required to implement

### PR DESCRIPTION
**Pull request by thijskh**
It was documented nowhere what endpoints your app needs to support and what they need to do.

### CODE REVIEW

File: config/packages/framework.yaml [hunk 1]
```diff
@@ -4,6 +4,8 @@ imports:
 # see https://symfony.com/doc/current/reference/configuration/framework.html
 framework:
   secret: '%secret%'
+  esi: false
```
Comment (parijke) [inline config/packages/framework.yaml:7]: esi and fragments are both false by default. Do we still want the configure them explicitely?

**Review by parijke — APPROVED** [review]

File: composer.lock
Comment (parijke) [inline composer.lock:1]: @thijskh whu is the composer.lock file changed? Looks unreleated

**Review by parijke — CHANGES_REQUESTED** [review]

File: composer.lock
Comment (thijskh) [reply composer.lock:1]: Probably because of the different base branches. Will rebase it

**Comment by thijskh** [comment]
Replaced by #6

**Pull request closed without merging**